src/components/AppHeaderLeft.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const AppHeaderLeft = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: space-around;
+    width: 25%;
+    height: 100%;
+    @media only screen and (max-width: 415px) {
```

Comment:
Same as in above comment, also 415px should be a variable.

Reply:
done